### PR TITLE
Adding AUD as @currency

### PR DESCRIPTION
**Pull request by ideabrian**
Our contact in Australia is requesting that we provide the invoice in AUD.

I added it, but would like a review.

### CODE REVIEW

File: app/views/account/InvoicesView.coffee [hunk 1]
```diff
@@ -62,12 +66,14 @@ module.exports = class InvoicesView extends RootView
     jqxhr.done =>
       application.tracker?.trackEvent 'Finished invoice payment',
         amount: @amount
+        currench: @currency # added 2.17.2016
```
Comment (differentmatt) [inline app/views/account/InvoicesView.coffee]: Typo: `currench`

**Comment by differentmatt** [comment]
We don't need the added date comments, because git will keep track of that for us.

**Comment by differentmatt** [comment]
How did you test this?

File: app/views/account/InvoicesView.coffee [hunk 2]
```diff
@@ -19,6 +19,8 @@ module.exports = class InvoicesView extends RootView
     super(options)
     @amount = utils.getQueryVariable('a', 0)
     @description = utils.getQueryVariable('d', '')
+    # adding @currency 2.17.2016
+    @currency = utils.getQueryVariable('c', 'usd') # added 2.17.2016
```
Comment (differentmatt) [inline app/views/account/InvoicesView.coffee]: We should uppercase USD, even if Stripe technically supports lowercase.

Comment (ideabrian) [inline app/views/account/InvoicesView.coffee]: Done.

File: app/views/account/InvoicesView.coffee [hunk 3]
```diff
@@ -62,12 +66,14 @@ module.exports = class InvoicesView extends RootView
...
       application.tracker?.trackEvent 'Finished invoice payment',
         amount: @amount
+        currency: @currency # added 2.17.2016
         description: @description
 
       # Show success UI
       @state = 'invoice_paid'
       @stateMessage = "$#{(@amount / 100).toFixed(2)} " + $.t('account_invoices.success')
       @amount = 0
+      @currency = '' # added 2.17.2016
```
Comment (differentmatt) [inline app/views/account/InvoicesView.coffee]: Don't set this to an invalid value after payment completes.  Just leave it as is.

**Comment by differentmatt** [comment]
invoices-view.jade needs to be updated as well:
```jade
          label.control-label(for="amount")
            span.spr Amount in
            span.spr= view.currency
            span currency
```
And en.coffee:
```coffeescript
  account_invoices:
    amount: "Amount in US dollars"
    amount_in: "Amount in"
    currency: "currency"
```

**Comment by differentmatt** [comment]
We also need to update the server to use the correct currency.  These client side changes only affect what we display.

**Comment by ideabrian** [comment]
Did you hear back from Stripe regarding the currency change in the call to
the server?

**Comment by differentmatt** [comment]
@ideabrian Yes, we need to make sure we're setting the currency on our server correctly.

How much is this task blocking you?

**Comment by ideabrian** [comment]
This isn't blocking anything.

**Pull request closed without merging**